Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -261,6 +261,7 @@
         <dependency>
             <groupId>commons-fileupload</groupId>
             <artifactId>commons-fileupload</artifactId>
+            <version>1.3.3</version>
```

Comment:
can we define this in CONNECT pom.xml? I think commons-fileupload is defined in Product pom.xml